package-lock.json

```diff
@@ -1,8372 +1,8 @@
 {
```

Comment:
No need to include this file in the commit. You changed no dependencies.

Reply:
I remember installing `tsconfig-paths` but from the looks of it, it was already a dependency in the first place. I'll drop the file from the PR.